scripts/import_data.py

```diff
@@ -0,0 +1,420 @@
+#!/usr/bin/env python3
+
+import argparse
+import ast
+import datetime
+import ipaddress
+import json
+import subprocess
+import sys
+import uuid
+
+try:
+    import mysql.connector
+    from mysql.connector.connection import MySQLCursor
+    import pandas as pd
+    import yaml
+    import xlrd  # used by pandas
+except ImportError:
+    print("The following modules are required:")
+    print("mysql-connector-python")
+    print("pandas")
+    print("PyYAML")
+    print("xlrd")
+    print("You can install them with: pip install {module name}")
+    sys.exit(1)
+
+
+DEFAULT_COLLECTION_VISIBILITY = "private"  # can be changed to "public" or "internal"
+DEFAULT_NAME_COLUMN = "Nazwa"
+
+
+def get_current_time() -> str:
+    """Returns the current time in YYYY-MM-DD-hh:mm:ss format"""
+    return datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+
+
+def get_database_host(container_name: str) -> str:
+    """
+    Tries to automatically find the database host ip by container name.
+    :param container_name: name of the container to find ip address of
+    :return: ip address of the container
+    """
+    try:
+        host = subprocess.run(
+            "sudo docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' " + container_name,
+            shell=True, check=True, capture_output=True, encoding="utf-8"
+        ).stdout
+
+        ipaddress.ip_address(host)
```

Comment:
`ValueError: '172.21.0.2\n' does not appear to be an IPv4 or IPv6 address`
We should do:
`ipaddress.ip_address(host.rstrip())`